scripts/test-harness.sh

```diff
@@ -174,6 +174,22 @@ while getopts "n:t:x:d:s:l:" opt; do
     l)  if valid_language_name "${OPTARG}"; then LANGUAGE=${OPTARG};
         else echo "Invalid language name ${OPTARG}" && help && exit 1;
         fi ;;
+    c)  if [[ ${OPTARG} -eq 1 ]]; then
+            if [[ "$LANGUAGE" == "c" ]]; then
+                TEST=($(cat list.def));
+                for tests in "${TEST[@]}"; do
+                         CTEST+=("micro-benchmarks/$tests")
+                 done
+                 TESTS=("${CTEST[@]}")
+            else
```

Comment:
What about "cpp"?? Are we checking ony c and Fortran?

Reply:
For now, only support C/C++ and Fortran.